tests/sync/test_codebook.py

```diff
@@ -0,0 +1,29 @@
+import unittest
+
+from utils import get_api_key
+
+import cohere
+
+API_KEY = get_api_key()
+co = cohere.Client(API_KEY)
+
+
+class TestCodebook(unittest.TestCase):
+    def test_success(self):
+        prediction = co.codebook(model="multilingual-22-12", compression_codebook="32x")
+        self.assertEqual(len(prediction.codebook), 96)
+        self.assertIsInstance(prediction.codebook[0], list)
+        self.assertIsInstance(prediction.codebook[1], list)
+        self.assertEqual(len(prediction.codebook[0]), 256)
+        self.assertEqual(len(prediction.codebook[0][0]), 8)
+        self.assertTrue(prediction.meta)
+        self.assertTrue(prediction.meta["api_version"])
+        self.assertTrue(prediction.meta["api_version"]["version"])
```

Comment:
same assertions apply for async test, and for default model test. right?
you can do something like
```python3
response_with_specified_codebook = co.codebook(model="multilingual-22-12", compression_codebook="32x")
response_without_specified_codebook = co.codebook(model="multilingual-22") # default
for response in [response_with_specified_codebook, response_without_specified_codebook]:
	# assert
	...
```

Reply:
updated!